llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -705,7 +707,25 @@ def fsdp_wrap_fn(self, module: nn.Module) -> bool:
 
     # Activation Checkpointing
     def activation_checkpointing_fn(self, module: nn.Module) -> bool:
-        return isinstance(module, MPTBlock)
+        if not hasattr(self.config, 'activation_checkpointing_target'):
+            return isinstance(module, MPTBlock)
+        act_ckpt_str = self.config.activation_checkpointing_target
+        act_ckpt_lst = act_ckpt_str.replace(' ', '').split(',')
```

Comment:
why is there a `.replace`?

Reply:
to remove space in the multi-name string, e,g 'grouped_query_attention,  MPTMLP'